wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

```diff
@@ -0,0 +1,278 @@
+2. The Qualifier pool contains 8 maps, all of which will use [FreeMod](/wiki/Gameplay/Game_modifier#freemod) rules, organised as such:
+   - Rice: 3 beatmaps
+   - LN 2 beatmaps
+   - Hybrid: 2 beatmaps
+   - SV: 1 beatmap
+3. Players will play the mappool at least once (and up to twice) at a designated time. Their best combined score, meaning the total of each beatmap's best score, will be the final result.
+4. The mappool will be played in the following order: SV1, RC1, RC2, RC3, LN1, LN2, HB1, HB2. Players may not ask to play the pool in any other order.
+   - Player may skip playing any beatmap on their second run.
+5. There will be an optional 5-minute break in between the first and second playthrough of the mappool.
+6. We suggest players not to broadcast or share their results to avoid seed manipulation.
+7. Qualifier stage results will be used for seeding and for choosing captains.
+   - Higher scores lead to a higher chance to be picked during the drafting phase. Players are eliminated if they do not get picked, or if they get too low of a score during Qualifers.
+8. Some best-performance players will advance to drafting phase.
+   - Disrupting the match by foul play or seed manipulation will result in disqualification, and any such players will be announced before the drafting phase takes place.
+9. Players will use same multiplayer rooms if they select same time to play the Qualifiers. It is prohibited to abort a match with the reason of a technical problem, in which case the score will be recorded as is.
+
+### Stage instructions
+
+1. Following the Qualifier stage, all players will participate in a phase to form teams, called the *drafting phase*. Both divisions will have independent drafting phases. All unpicked players will be eliminated from the tournament.
+2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.
+3. Following the regular stage, the elimination stage will be played, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
+4. All losing teams in the loser bracket will be eliminated from the tournament and the last team will be the winner.
+
+### Drafting phase instructions
+
+1. The host will declare one captain for each team 24 hours before the drafting phase.
+   - Drafting order will also be declared by the host.
+   - Captains are selected by performance and preference.
+   - Non-captains will join a group of players waiting to join a team, herein called a "player pool".
+2. Captains decide their team members during the drafting phase.
+   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms. 
```

Comment:
Untranslated, but dosen't matter i guess. this is just an reminder for captains.

Reply:
fixed in 689aebb